tests/integration/test_analysisd/test_eps/test_no_eps_configuration.py

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
It may be redundant, but this test checks that the eps configuration is not configured and then check that the events received are similar to the events processed.
On the other hand, test_disabled just checks that the eps are disabled.